featuretools/primitives/standard/transform/time_series/__init__.py

```diff
@@ -23,3 +23,11 @@
 from featuretools.primitives.standard.transform.time_series.rolling_trend import (
     RollingTrend,
 )
+from featuretools.primitives.standard.transform.time_series.expanding import (
```

Comment:
If we are going to create a new folder for the expanding primitives, I think we should be consistent and do the same thing with our rolling* primitives so we have a consistent organizational structure.

That could be a separate small MR.

Reply:
I agree